prover/src/circuits/summa_circuit.rs

```diff
@@ -112,6 +112,13 @@ pub mod summa_hyperplonk {
                 grand_total,
             }
         }
+
+        pub fn init_empty() -> Self {
```

Comment:
Is this method used anywhere?

Reply:
Thanks for pointing that out! I almost forgot to update its usage:

https://github.com/summa-dev/summa-solvency/blob/15374914a63878e92e7ee0f332ca2634496fef01/backend/examples/summa_solvency_flow.rs#L120-L126

This is how it can be used.